trame_vtk/modules/vtk/__init__.py

```diff
@@ -1,6 +1,7 @@
 import warnings
 
 from .core import HybridView
+from .protocols.serializers import mesh as vtk_mesh
```

Comment:
Should be `from .serializers import ...`

Reply:
Done